modules/rosa-getting-started-configure-an-idp.adoc

```diff
@@ -8,6 +8,11 @@
 
 You can configure different identity provider types for your {product-title} (ROSA) cluster. Supported types include GitHub, GitHub Enterprise, GitLab, Google, LDAP, OpenID Connect and HTPassword identity providers.
 
+[IMPORTANT]
+====
+The `htpasswd` identity provider option is included only to enable the creation of a single, static administration user. `htpasswd` is not supported as a general use identity provider for {product-title}.
```

Comment:
Same comments as for [config-idp.adoc](https://github.com/openshift/openshift-docs/pull/42822/files#diff-35db2c8995815f7c466cd4dab6f0c13698b5e30a7e8c9b04e09f843c1dfea76b)

Reply:
Thank you. I have resolved those points throughout the PR.